modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMHomePage.testcase

```diff
@@ -49,4 +49,41 @@ definition {
 		}
 	}
 
+	@description = "LPS-172079 - Verify that the admin user can add a new announcement."
+    @priority = "5"
+    test AdminCanAddAnnouncement {
+
+        task ("When: The User goes to announcements in web content"){
+
+        WebContentNavigator.openWebContentAdmin(siteURLKey = "partner-portal");
+
+        PRMUtils.gotoFolder(folderName = "Announcements");
+        }
+
+        task("Then: User add an announcement and publish it"){
```

Comment:
Hey @marinaasant0s, I think that we can improve this task.
It could be: "When: User add a new announcement and publish it"